scripts/ci/jobs/check-konflux-pipelines.sh

```diff
@@ -24,11 +26,45 @@ To resolve:
 
 ${expected_runafter}
         "
-        exit 1
+        return 1
+    else
+        echo "✓ No diff detected."
+    fi
+}
+
+check_all_components_part_of_custom_snapshot() {
+    local pipeline_path=".tekton/operator-bundle-pipeline.yaml"
+    local task_name="create-acs-style-snapshot"
+
+    actual_components="$(yq eval '.spec.tasks[] | select(.name == '\"${task_name}\"') | .params[] | select(.name == "COMPONENTS") | .value' "${pipeline_path}" | yq eval '.[].name' - | tr " " "\n" | sort)"
```

Comment:
I'm curious what role does `tr " " "\n"` play in this and below commands?

Reply:
`yq eval '.[].name'` emits a string like `central-db main operator`. I found it most concise to use `tr " " "\n"` to turn it into multi-line string instead of  attempting to use bash arrays, because sort and diff work properly. 
I will add a comment.

Reply:
I asked because for me the command was returning items printed one per line:
![image](https://github.com/user-attachments/assets/a2a8f010-7c82-431d-ad23-c1f18d90b22b)

`yq` version is 4.30.8.

Let's stay with `tr` if it provides portability but it's overall worrying that different versions of `yq` work so differently.

Reply:
huh, that is in fact worrying.